mdoc/Test/en.expected.typeForwards/TheNamespace/TheClass.xml

```diff
@@ -12,6 +12,17 @@
   <AssemblyInfo>
     <AssemblyName>DocTest-typeForwards-Second-First</AssemblyName>
   </AssemblyInfo>
+  <AssemblyInfo>
+    <AssemblyName>DocTest-typeForwards-Third</AssemblyName>
+    <AssemblyVersion>0.0.0.0</AssemblyVersion>
+  </AssemblyInfo>
+  <AssemblyInfo>
+    <AssemblyName>DocTest-typeForwards-Third-First</AssemblyName>
+  </AssemblyInfo>
+  <TypeForwardingChain>
+    <TypeForwarding From="DocTest-typeForwards-Second-First" FromVersion="0.0.0.0" To="DocTest-typeForwards-Second" ToVersion="0.0.0.0" />
```

Comment:
Isn't this the type forward for the framework `Two` ? This FrameworkAlternate attribution doesn't seem correct. If I'm remembering this test correctly, I would have expected to see `FrameworkAlternate="Two"` on this one, and `FrameworkAlternate="Three"` on the following line ... right?

Reply:
To make sure the test case will cover removing "FrameworkAlternate" attribute if it includes all frameworks, I edited "make" file to add type forwarding for framework `One` as well.

```
cp Test/DocTest-typeForwards-Second-First.dll Test/FrameworkTestData-fx-typeForwards/One
cp Test/DocTest-typeForwards-Second.dll Test/FrameworkTestData-fx-typeForwards/dependencies/One
```

Reply:
I think this is still not right ... the `One` framework shouldn't have this assembly (as it kind of negates the entire test, which is meant to represent a type being introduced in one version, then being moved into another assembly in a future release).

If you remove this most-recent change (so that the type forward exists only in `Two` and `Three`) ... then I would expect to see the following in the XML:

```xml
<TypeForwarding From="DocTest-typeForwards-Second-First" FromVersion="0.0.0.0" To="DocTest-typeForwards-Second" ToVersion="0.0.0.0" FrameworkAlternate="Two />
<TypeForwarding From="DocTest-typeForwards-Third-First" FromVersion="0.0.0.0" To="DocTest-typeForwards-Third" ToVersion="0.0.0.0" FrameworkAlternate="Three" />
```
Which says, "in moniker `Two`, it's forwarded from X to Y, and in moniker `Three` it's forwarded from moniker Z to N" ... of course, if a type is forwarded in *every moniker*, then I would expect to see the `TypeForward` tag but with no `FrameworkAlternate` attribute, because at that point the list of frameworks will be all of them :)